src/app/containers/MostRead/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Canonical from './Canonical';
+
+const getLocalMostReadEndpoint = (service, variant) => {
+  const localhostURL = 'http://localhost:7080';
```

Comment:
For context for others reviewing this PR - we're keeping the  fine to keep this here for now, since we're awaiting having an endpoint available on Test / Live that we can hit directly.